src/packet_analysis/protocol/unknown_ip_transport/UnknownIPTransport.h

```diff
@@ -23,10 +23,7 @@ class UnknownIPTransportAnalyzer final : public IP::IPBasedAnalyzer {
     packet_analysis::IP::SessionAdapter* MakeSessionAdapter(Connection* conn) override;
 
 protected:
-    /**
-     * Parse the header from the packet into a ConnTuple object.
-     */
-    bool BuildConnTuple(size_t len, const uint8_t* data, Packet* packet, ConnTuple& tuple) override;
+    bool InitConnKey(size_t len, const uint8_t* data, Packet* packet, IPBasedConnKey& key) override;
```

Comment:
Perhaps keep the comment? (I know, the other functions don't have it).

Reply:
I've left it out. Agree on verbose comments on the API/interface, but think it's okay to be more lenient for actual implementations of a given API - it'd just be a repeat.

There's doxygen's `@copydoc` - maybe we could have a discussion of using that more often, though that'll only make sense when looking at the generated HTML and I realize we don't actually have a doxygen setup :-(